packages/http-urlencode-path-parser/__tests__/index.js

```diff
@@ -0,0 +1,25 @@
+const middy = require('../../core')
+const urlEncodePathParser = require('../')
+
+describe('📦 Middleware URL Encoded Path Parser', () => {
+  test('It should decode simple url encoded requests', () => {
```

Comment:
I would add test for bad case as well here

Reply:
Added in the example from #429